client/app/bundles/course/lectures.js

```diff
@@ -0,0 +1,33 @@
+$(document).ready(() => {
+  /* global Routes */
+  $(document).on('click', '.gen-access-link', function() {
+    const $this = $(this);
+    const splits = $this.attr('id').split('-');
+    const courseId = splits[1];
+    const lectureId = splits[2];
+    $this.addClass('disabled');
+    $.get(Routes.access_link_course_lecture_path(courseId, lectureId, { format: 'json' }),
```

Comment:
😄 Thanks for moving this piece of code to the client folder.  But we have certain conventions here also:

All of the client side APIs should be defined and documented in https://github.com/Coursemology/coursemology2/tree/master/client/app/api

( this is quite new, so that we have some old code which stills directly calls the server API and they will be cleaned up finally. so let's follow the convention and move new code there )